docs/user_guide/hopsworks/dataSetShare.rst

```diff
@@ -1,21 +1,84 @@
-===========================
-Share a Data Set
-===========================
-
-Only a `data owner` or the `project owner` has privileges to share
-Data Sets. To share a Data Set, go to the `Data Sets Browser` in your
-project, and right-click on the Data Set to be shared and then select
-the ``Share DataSet`` option. A popup dialog will then prompt you to
-select (1) a target project with which the *Data Set* is to be Shared
-and whether the *Data Set* will be shared as read-only (**Can View**)
-or as read-write (**Can edit**). To complete the sharing process, a
-Data Owner in the target project has to click on the shared Data Set,
-and then click on ``Acccept`` to complete the process.
-
-.. figure:: ../../imgs/share-dataset.png
-    :alt: Share dataset
-    :scale: 100
+===================================
+Data Set sharing and access-control
+===================================
+
+
+Share
+-----
+
+The basic principle of the Project - Data Set abstraction is that no data can be copied/moved outside of a
+project's scope, making a project effectively a sandbox for data and programs. However, Data Sets can be shared with one
+or more other projects and only `data owners` have privileges to share Data sets. To share a Data Set, navigate to
+the ``Data Sets`` browser from the services menu on the left, right-click on the Data Set to be shared and from the
+``Share with`` option select either ``project`` or ``cluster``.
+
+.. _share-ds-1.png: ../../_images/share-ds-1.png
+.. figure:: ../../imgs/share-ds-1.png
+    :alt: Share a Data Set
+    :target: `share-ds-1.png`_
+    :scale: 25%
+    :align: center
+    :figclass: align-center
+
+In the case of ``project``, a popup dialog will then prompt users to select a target project with which the *Data
+Set* is to be share with. It is also feasible to filter projects by name as shown in the image below.
+
+.. _share-ds-2.png: ../../_images/share-ds-2.png
+.. figure:: ../../imgs/share-ds-2.png
+    :alt: Project to share the Data Set with
+    :target: `share-ds-2.png`_
+    :scale: 25%
+    :align: center
+    :figclass: align-center
+
+
+To complete the sharing process, a Data Owner in the target project has to click on the shared Data Set,
+and then click on ``Acccept`` to complete the process, ``Reject`` to reject the shared Data Set or ``cancel`` for no
+action.
+
+.. _share-ds-3.png: ../../_images/share-ds-3.png
+.. figure:: ../../imgs/share-ds-3.png
+    :alt: Accept a shared Data Set
+    :target: `share-ds-3.png`_
+    :scale: 30%
     :align: center
     :figclass: align-center
 
-    Share dataset
+Unshare
+-------
+
+Data Sets can be ``unshared`` by `data owners`, either from the parent project or from the project they have been
+shared with. To do that, users of the parent project can right-click on the Data Set, click ``UnShare from`` and then
```

Comment:
`unshared` or `UnShare`?

Reply:
I'm copying whatever is in the UI , which is ``UnShare from``. In another jira we can fix the UI